### PR TITLE
Remove npm-check dependency

### PR DESCRIPTION
**Pull request by jzolnowski**
Fix the [Got allows a redirect to a UNIX socket](https://github.com/xlts-dev/angularjs-webpack-babel/security/dependabot/40) security issue

### CODE REVIEW

File: package-lock.json
Comment (gkalpak) [inline package-lock.json:1]: We shouldn't be changing the lockfile version in this PR.

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Comment by jzolnowski** [comment]
I updated the commit message, generated package-json without changing `lockfileVersion`, replaced `npm-check` with [npm-check-updates](https://github.com/raineorshine/npm-check-updates). At this point, it doesn't have any [vulnerabilities](https://security.snyk.io/package/npm/npm-check-updates)

File: package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "test:e2e:debug": "playwright test --project chromium --headed",
     "test:e2e:report": "npx playwright show-report",
     "test": "run-s lint test:unit:single test:e2e",
-    "update": "npm-check -u",
+    "update": "ncu -u",
```
Comment (gkalpak) [inline package.json]: If I am not mistaken, `ncu -u` only updates `package.json`, but it does not install the updated versions (and most importantly it does not update the lockfile). Should we manually install:

```suggestion
    "update": "ncu -u && npm install",
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for adding info and links in the commit message. However, it seems that the commit message header and body have been merged into one (which is not the correct format).

I would expected something like:

> chore(package): replace \`npm-check\` devDependency with \`npm-check-updates\`
>
> Replace \`npm-check\` devDependency with [npm-check-updates][1] due to the
> [got allows a redirect to a unix socket][2] security issue.
>
> [1]: https://github.com/raineorshine/npm-check-updates
> [2]: https://github.com/xlts-dev/angularjs-webpack-babel/security/dependabot/40

File: package.json (hunk 1 above)
Comment (jzolnowski) [reply package.json]: Updated

**Comment by jzolnowski** [comment]
Commit message updated, ready for another round 😄

**Review by gkalpak — APPROVED** [review]
The changes lgtm :+1:

(The commit message body is in a funny format, but 🤷‍♂️ :grin:)

**Pull request merged**